sw/device/lib/crypto/aes_gcm/aes_gcm.c

```diff
@@ -145,7 +148,7 @@ static inline uint8_t block_byte_get(const aes_block_t *block, size_t index) {
  * @return Offset of end of buffer from last full block
  */
 static inline size_t get_block_offset(size_t sz) {
-  return sz & (kAesBlockNumBytes - 1);
+  return sz & ((1 << kAesBlockLog2NumBytes) - 1);
```

Comment:
Nit: the comment above about `kAesBlockNumBytes` being a power of two is a little stale after replacing `kAesBlockNumBytes` with `1 << kAesBlockLog2NumBytes`.